test/Microsoft.Azure.WebJobs.Host.UnitTests/Loggers/WebJobsSanitizingInitializerTests.cs

```diff
@@ -11,6 +11,9 @@ public class WebJobsSanitizingInitializerTests
     {
         private const string StorageString = "DefaultEndpointsProtocol=https;AccountName=testaccount;AccountKey=testkey;";
         private const string SecretReplacement = "[Hidden Credential]";
+        private const string AssemblyLoadErrorWithAllowedToken = "System.AggregateException: aggregate error --->System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.Azure.WebJobs.Host, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' or one of its dependencies.The system cannot find the file specified.at System.Reflection.RuntimeAssembly._nLoad(AssemblyName fileName, String codeBase, Evidence assemblySecurity, RuntimeAssembly locationHint, StackCrawlMark & stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)";
+        private const string AssemblyLoadError = "System.AggregateException: aggregate error --->System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.Azure.WebJobs.Host, Version=2.0.0.0, Culture=neutral, Token=31bf3856ad364e35' or one of its dependencies.The system cannot find the file specified.at System.Reflection.RuntimeAssembly._nLoad(AssemblyName fileName, String codeBase, Evidence assemblySecurity, RuntimeAssembly locationHint, StackCrawlMark & stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)";
```

Comment:
Can you make this one have both `PublicKeyToken=` and `Token=`? We want to make sure it replaces `Token=` and keeps the allowed one.